docs/content/docs/operations/deployment.md

```diff
@@ -24,18 +24,141 @@ under the License.
 
 ## Local Run with Test Data
 
-{{< hint warning >}}
-**TODO**: How to run with test data with LocalExecutorEnvironment.
-{{< /hint >}}
+After completing the [installation of flink-agents]({{< ref "docs/get-started/installation" >}}) and [building your agent]({{< ref "docs/development/workflow_agent" >}}), you can test and execute your agent locally using a simple Python script. This allows you to validate logic without requiring a Flink cluster.
+
+### Key Features
+
+- **No Flink Required**: Local execution is ideal for development and testing.
+- **Test Data Simulation**: Easily inject mock inputs for validation.
+- **IDE Compatibility**: Run directly in your preferred development environment.
+
+### Data Format
+
+#### Input Data Format
+
+The input data should be a list of dictionaries (`List[Dict[str, Any]]`) with the following structure:
+
+``````python
+[
+    {
+        # Optional field: Context key for multi-session management
+        # Priority order: "key" > "k" > auto-generated UUID
+        "key": "session_001",  # or use shorthand "k": "session_001"
```

Comment:
This is hard to understand. What is multi-session management? What is auto-generated UUID? We really need to keep it simple, and should not confuse users with unnecessary complexity.